t/kubernetes/discovery/kubernetes2.t

```diff
@@ -749,3 +764,26 @@ $::scale_ns_c",
     "DONE\n",
     "{ 0 0 1 0 0 1 }\n",
 ]
+
+
+
+=== TEST 12: parse informer_factory information
+--- yaml_config eval: $::yaml_config
+--- request eval
+[
+
+"GET /queries
+[
+  \"first/ns-a/ep:p1\",\"first/ns-a/ep:p2\",
+  \"second/ns-a/ep:p1\",\"second/ns-a/ep:p2\"
+]",
+
+"GET /informer
+[]"
+
+]
+--- response_body eval
+[
+    "{ 0 0 0 0 }\n",
+    "{ 0 }\n",
+]
```

Comment:
The test case seems meanless?

Reply:
exactly, I struggle to make any sense out of it.

Reply:
+1.
The test case should indicate that using a namespace based informator can properly watch objects.

Reply:
Will remove it.